src/Compilers/VisualBasic/Portable/Symbols/UsedAssemblies.vb

```diff
@@ -0,0 +1,211 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.PooledObjects
+Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
+Imports Microsoft.CodeAnalysis.VisualBasic.Symbols.Retargeting
+
+Namespace Microsoft.CodeAnalysis.VisualBasic
+
+    Partial Public Class VisualBasicCompilation
```

Comment:
Please rename this file `VisualBasicCompilation_UsedAssemblies.cs` to keep inline with other partial class naming conventions. I also find it very odd and confusing that this file is not in the Compilation folder.

Reply:
Actually, it looks like our VB sources disagree with the C# conventions here. I find this version confusing, but ok.

---
In reply to: [369249047](https://github.com/dotnet/roslyn/pull/40987#discussion_r369249047) [](ancestors = 369249047)